### ISSUE TITLE
defining services

### ISSUE THREAD

**Issue by dark2201** [amzn/ion-schema-kotlin#74]
it would be cool if ion schema supported defining services like protobuf https://developers.google.com/protocol-buffers/docs/proto#services

**Comment by pbcornell** [comment]
Yes, agreed, that is a cool idea!  If you have any specifics in mind (or aspects of protobuf services you'd like to see improved), feel free to share them here.  Thank you
